Godeps/_workspace/src/k8s.io/kubernetes/test/e2e/kubectl.go

```diff
@@ -755,7 +755,7 @@ var _ = Describe("Kubectl client", func() {
 		var rcPath string
 		var nsFlag string
 		containerName := "redis-master"
-		BeforeEach(func() {
+		JustBeforeEach(func() {
```

Comment:
yes, I'm sweeping through the upstream tests as well to see if there are any more that would need changing